grovedb/src/lib.rs

```diff
@@ -250,7 +250,7 @@ impl GroveDb {
     }
 
     /// Deletes GroveDB folder from disk and drop GroveDB instance
```

Comment:
all good but a comment needs to be updated

Reply:
got it thanks.